Makefile

```diff
@@ -163,10 +174,19 @@ build-dirs:
 	@mkdir -p bin/$(ARCH)
 	@mkdir -p .go/src/$(PKG) .go/pkg .go/bin .go/std/$(ARCH)
 
-clean: container-clean bin-clean
+clean: # @HELP cleans the repository's temporary files
+clean: container-clean bin-clean 
 
 container-clean:
 	rm -rf .container-* .dockerfile-* .push-*
 
 bin-clean:
 	rm -rf .go bin
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?# @HELP .*$$' $(MAKEFILE_LIST) \
```

Comment:
regex should include 0-9.  Really probably should just be `^.*: *# *@HELP ` in fact